rotkehlchen/api/v1/schemas.py

```diff
@@ -3111,6 +3111,11 @@ def transform_data(
                     f'Given value {data["address"]} is not a valid {data["blockchain"]} address',
                     field_name='address',
                 ) from e
+        elif data['blockchain'] is None and data['address'].startswith('0x'):
+            try:  # if the address looks like an EVM address try to checksum it
+                address = to_checksum_address(data['address'])
```

Comment:
use `deserialize_evm_address` here and above. 

also, why not use `is_hex_address()` instead `data['address'].startswith('0x')` ?